README.md

```diff
@@ -93,6 +93,25 @@ end
   end
 ```
 
+- Last line of a multiline array or hash should end with a trailing comma. It keeps diffs much smaller when adding or deleting lines in future.
```

Comment:
> keeps diffs much smaller when adding or deleting lines in future

The benefit is always 2 line fewer diff if I'm not wrong (see diffs below). It _does_ optimise the diff output I admit, however IMO not really significantly. Also I find array/hash followed by trailing comma a bit against my instinct because personally when I see comma, I would expect another item following up.

I'd suggest our style guide optimising for human brains rather than cvs tools. :smile: 

```diff
@@ -2,7 +2,5 @@ diff_hash = {
   a: 1,
   b: 2,
   c: 3,
-  d: 4,
-  e: 5,
-  f: 6
+  d: 4
 }
```

vs

```diff
@@ -3,6 +3,4 @@ diff_hash = {
   b: 2,
   c: 3,
   d: 4,
-  e: 5,
-  f: 6,
 }
```